website/client/components/groups/group.vue

```diff
@@ -546,15 +546,14 @@ export default {
         eventLabel: 'Start a Quest',
       });
 
-      let hasQuests = find(this.user.items.quests, (quest) => {
+      let hasQuests = this.user.items.quests.some((quest) => {
```

Comment:
`user.items.quests` is an object, not an array, so `some` isn't available as a methods

Reply:
I was arguing my way through the best solution when I found out the entire method isn't actually used. Apparently whoever extracted the quest sidebar from the group page forgot to remove it. 

Have to say the result for users that have no quests is sort of ugly, though. The quest modal would definitely benefit from a default image, maybe a big question mark? Anyway, feels out of scope, so I just removed the function and everything seems dandy.